src/server/app.py

```diff
@@ -16,11 +16,17 @@ def postAddition():
     numbers = []
     
     for token in operands.split(','):
-        numbers.append(int(token))
+        if token=="":
+            numbers.append("blank")
+            break
```

Comment:
Your solution breaks in the case of the following input: `"3,,10"` 

If a token does come up to be blank (`operands.split()` would yield `["3", "", "10"]`) then why not skip that token rather than insert it into the number's list